<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDlg</class>
 <widget class="QDialog" name="LoginDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hi</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>46</y>
     <width>135</width>
     <height>129</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="remoteHostIpAddressLineEdit">
      <property name="text">
       <string>localhost</string>
      </property>
      <property name="placeholderText">
       <string>主机地址</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="remoteHostPortLineEdit">
      <property name="text">
       <string>23333</string>
      </property>
      <property name="placeholderText">
       <string>端口号码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="usrLineEdit">
      <property name="text">
       <string>Infinity</string>
      </property>
      <property name="placeholderText">
       <string>尊姓大名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pswlineEdit">
      <property name="text">
       <string>233</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>登录密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="loginButton">
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
